flake.nix

```diff
@@ -31,6 +31,17 @@
             rust-analyzer
             cargo-outdated
 
+            # wlcs
+            (writeScriptBin "wlcs" ''
+              #!/bin/sh
+              ${wlcs}/libexec/wlcs/wlcs "$@"
+            '')
+
+            # build time stuff
+            pkg-config
+            protobuf
+            luarocks
```

Comment:
```suggestion
```

pkg-config was moved to native build deps and the other two are below

Reply:
And done